lib/array/index.js

```diff
@@ -0,0 +1,11 @@
+export function flatten(array) {
+  return Array.prototype.concat.apply([], array);
+}
+
+export function flatMap(array, lambda) {
+  return flatten(array.map(lambda));
+}
+
+export function uniq(array) {
```

Comment:
They were so small it was quicker to write them rather than break flow and look at the docs for ramda. I can switch them out now I've finished.